api/src/main/java/org/openmrs/api/impl/UserServiceImpl.java

```diff
@@ -766,20 +768,22 @@ public User setUserActivationKey(User user) {
 		credentials.setActivationKey(activationKey);	
 		dao.setUserActivationKey(credentials);	
 		
-		String link = Context.getAdministrationService().getGlobalProperty(OpenmrsConstants.GP_HOST_URL)
-		        .replace("activationkey", "activationkey=" + token);
-		String msg = "Hi " + user.getUsername()
-		        + ",\n It looks like you want to change your password, click on the following link to do so.\n"
-		        + link
-		        + "\n Please disregard this email if you didn't innitiate the password reset process.  \n The openmrs Team";
+		Properties props = new Properties();
+		InputStream stream = OpenmrsClassLoader.getInstance().getResourceAsStream("emailTemplate.properties");
```

Comment:
The file isn't supposed to be a properties file, otherwise you might as well move everything to the messages.properties file which already supports internationalization anyways and get rid of this file.